homeassistant/components/ihc/__init__.py

```diff
@@ -64,6 +66,8 @@ def validate_name(config):
 
 LIGHT_SCHEMA = DEVICE_SCHEMA.extend({
     vol.Optional(CONF_DIMMABLE, default=False): cv.boolean,
+    vol.Optional(CONF_OFF_ID, default=0): cv.positive_int,
```

Comment:
Is this only applicable to light actuators?

Reply:
Good point, as this is also relevant for switch domain, so I will add that.